.ddev/commands/host/clone

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 
 ## Description: Checkout a module using git clone
-## Usage: clone MODULE_NAME [-b BRANCH_NAME | -t TAG_NAME] [-c]
-## Example: "ddev clone admin_toolbar" or "ddev clone admin_toolbar -b develop -c"
+## Usage: clone MODULE_NAME [-b BRANCH_NAME | -t TAG_NAME] [-c] [-o ORIGIN_PATH]
+## Example: ddev clone admin_toolbar\nddev clone my_module -b develop -o git@github.com:palantirnet
```

Comment:
Typo

Reply:
The `\n` forces the next example to be on the next line. Could you review it with `ddev help clone` and see if it looks correct?